src/components/Common/PaginationControl/PaginationControl.tsx

```diff
@@ -47,6 +49,7 @@ const DefaultPaginationControl = ({
           </section>
         </div>
         <div className={style.paginationControlButtons}>
+          {isFetching && <InlineSpinner ariaLabel={t('labels.paginationFetchingLabel')} />}
```

Comment:
Nit: we'll potentially need to adapter this since it's not using the core loading indicator. No need to do that now, but we'll probably need to provide that for adapter customers down the road